CANNOT build 'create-gossipstore.c' because of pointer-size mismatch.

I was following c-Lightning github development progress and build it sometimes. But, I encountered a new error and can't go ahead.
I can understand compiler says what the problem is. (sizeof(pointer variable) and fscanf() format mismatch).
But, I don't know why it happened suddenly and What I should do?

```
Error message:
---------------------------------------------------------------------------------------
cc -DBINTOPKGLIBEXECDIR='"'../libexec/c-lightning'"' -Werror -Wall -Wundef -Wmissing-prototypes -Wmissing-declarations -Wstrict-prototypes -Wold-style-definition -std=gnu11 -g -fstack-protector  -I ccan -I external/libwally-core/include/ -I external/libwally-core/src/secp256k1/include/ -I external/jsmn/ -I external/libbacktrace/ -I external/libbacktrace-build -I external/libsodium/src/libsodium/include -I external/libbase58/ -I . -I/usr/local/include    -DSHACHAIN_BITS=48 -DJSMN_PARENT_LINKS  -DCOMPAT_V052=1 -DCOMPAT_V060=1 -DCOMPAT_V061=1 -DCOMPAT_V062=1 -DCOMPAT_V070=1 -DBINTOPKGLIBEXECDIR='"'../libexec/c-lightning'"'  -c -o devtools/create-gossipstore.o devtools/create-gossipstore.c
devtools/create-gossipstore.c: In function 'load_csv_file':
devtools/create-gossipstore.c:44:36: error: format '%ld' expects argument of type 'long int *', but argument 4 has type 'u64 * {aka long long unsigned int *}' [-Werror=format=]
         while(fscanf(scidf, "%s ,%ld\n", str, &scidsats[i].sat.satoshis) == 2 ) { /* Raw: read from file */
                                  ~~^          ~~~~~~~~~~~~~~~~~~~~~~~~~
                                  %lld
cc1: all warnings being treated as errors
<builtin>: recipe for target 'devtools/create-gossipstore.o' failed
make: *** [devtools/create-gossipstore.o] Error 1
---------------------------------------------------------------------------------------
```

How to reproduce:

```
sudo apt-get update
sudo apt-get install -y \
  autoconf automake build-essential git libtool libgmp-dev \
  libsqlite3-dev python python3 net-tools zlib1g-dev libsodium-dev
git clone https://github.com/ElementsProject/lightning.git
cd lightning
./configure
make
```
Additional Information:

```
$ getconf LONG_BIT
32
$ uname -m
armv7l
$ uname -i
armv7l
$ uname -a
Linux odroid 4.14.107-157 #1 SMP PREEMPT Thu Mar 21 09:59:50 -03 2019 armv7l armv7l armv7l GNU/Linux
```

--- a/devtools/create-gossipstore.c
+++ b/devtools/create-gossipstore.c
@@ -41,7 +41,7 @@ static struct scidsat *load_csv_file(FILE *scidf)
 	if (r != 2 || strcmp(title, "scid") != 0 || strcmp(&title[6], "satoshis") != 0)
 		err(1, "reading 'scid ,satoshis' from csv failed");
 
-        while(fscanf(scidf, "%s ,%ld\n", str, &scidsats[i].sat.satoshis) == 2 ) { /* Raw: read from file */
+        while(fscanf(scidf, "%s ,%"SCNu64"\n", str, &scidsats[i].sat.satoshis) == 2 ) { /* Raw: read from file */
 		if (!short_channel_id_from_str(str, strlen(str), &scidsats[i].scid, 0))
 			err(1, "failed to make scid struct");
 		i++;

